community/github/issues.md

```diff
@@ -0,0 +1,82 @@
+# pyOpenSci GitHub issue process
+
+When possible, pull request and issue submissions and reviews should follow
+standard open source workflows. Below are guidelines for handling issues.
+
+## Guidelines for New Issues
+
+
+**Issues in should be as specific as possible:** specifics within an issue help both our future selves and also outside contributors understand the goal or
+desired outcome associated with addressing the issue. This is important both internally and for issues that we tag as `help-wanted`, which we hope community members will address in pyOpenSci sprints.
+
+To ensure an issue is well-written and specific, include the following details:
+
+- **Clear Title:** A concise and descriptive title summarizing the issue or feature request helps us scan through issues and understand what each issue is about. Below are some example titles that are specific
+
+    * `Bug: broken link in link-to-page-here / name of page / document etc`
+    * `Fix: confusing paragraph on Python packaging with Hatch`
+    * `Add: page on using pixi for dependencies`
+
+- **Description:** A detailed explanation of the issue or feature request, including context, background information, and the reason for the request. Explain why the issue is important and what problem it solves.
+
+- **Screenshots/Code Samples:** Screenshots, code snippets, links, or any other relevant files that can help in understanding the issue better.
+
+- **Possible Solutions/Recommendations:** Any ideas or suggestions for how to address the issue, which can help guide contributors towards a solution.
+
+- **Related Issues or Pull Requests:** References to any related issues or pull requests, providing additional context and understanding of the broader scope of the issue.
+
+### If you are reporting a code bug
+
+* **Steps to Reproduce (for bugs):** A step-by-step guide on how to reproduce the issue, including relevant code snippets, commands, or configurations.
+
+* **Expected vs. Actual Behavior (for bugs):** A description of what you expected to happen and what actually happened. This helps in understanding the discrepancy and the impact of the bug.
+
+* **Environment Details (for bugs):** Details about the environment where the issue was observed, such as operating system, Python version, library versions, and any other relevant software/hardware details.
+
+### If you have permissions, label the issue
+
+While outside contributors will not have permission to label issues, pyOpenSci core team members and volunteers will. Be sure to add appropriate labels to
```

Comment:
it would be helpful to have a quick list here on what our labels are and what they're used for

Reply:
we have an open help-wanted ticket on the labels. the labels are different in each repo. 

https://github.com/orgs/pyOpenSci/projects/3?pane=issue&itemId=62313858

so while this is a good idea it's a lot more complicated than a quick list! we will need to complete the task abbove which involved deciding what labels we want to curate and syncing them across repos! jon did a bunch of work on this at our scipy sprint. you can open a new issue about this here that will be a future issue when we get that workflow setup  if you wish.